airflow/configuration.py

```diff
@@ -403,6 +403,52 @@ def inversed_deprecated_sections(self):
     upgraded_values: dict[tuple[str, str], str]
     """Mapping of (section,option) to the old value that was upgraded"""
 
+    legacy_incompatible_defaults: dict[tuple[str, str], tuple[Pattern, str, str]] = {
+        ("logging", "log_filename_template"): (
+            re.compile(
```

Comment:
fwiw, I took the default out of a fresh 2.10.5 config file, and this regex did _not_ match against it. The regex may have been cut off at the end of the task_id line (that's naively what printed, at least).

Give that a test.